src/v0/destinations/topsort/data/TopsortItemConfig.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "destKey": "properties.position",
+    "sourceKeys": "position",
+    "required": false
+  },
+  {
+    "destKey": "properties.product_id",
+    "sourceKeys": "productId",
+    "required": false
+  }
+]
```

Comment:
This `destKey` and `sourceKey` are incorrect it should be opposite. Also this mapping doesn't take into consideration the `product` context. A product object doesnt have `properties.position` but it directly has `prosition` where as in rudder message it has `properties.position` you need add array in `sourceKeys` to handle it